### PR TITLE
feat: Add .travis.yml

### PR DESCRIPTION
**Pull request by Mikhail-Sosnin**

### CODE REVIEW

**Review by svitkovsergey — APPROVED** [review]
Nicely done! 👍

**Review by mnesvit — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+language: scala
+
+scala:
+  - 2.12.9
+
+script:
+  - sbt clean compile test
```
Comment (AlekseiLitkovetc) [inline .travis.yml]: Is it possible to perform integration tests too?
`sbt clean compile testAll`

It will require docker.

Comment (Mikhail-Sosnin) [reply .travis.yml]: @AlekseiLitkovetc  added

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+language: scala
+
+scala:
+  - 2.12.9
+
+script:
+  - sbt clean compile test it:test
```
Comment (AlekseiLitkovetc) [inline .travis.yml]: We've already added alias for `test it:test`. 
For convenience you could use it: `sbt clean compile testAll`

Comment (AlekseiLitkovetc) [reply .travis.yml]: Also before we merge this PR we need to return one dependency after previous PR.
@likeanowl could we return `testContainers `?

Thank you in advance!

**Review by DenisVerkhoturov — APPROVED** [review]

**Pull request merged**